jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/AddRemoveLogs.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import com.intellij.execution.util.ListTableWithButtons
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.keymap.KeymapUtil
+import com.intellij.ui.CommonActionsPanel
+import com.intellij.ui.SimpleTextAttributes
+import com.intellij.ui.TableUtil
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.StatusText
+
+abstract class AddRemoveLogs<T>(emptyTableMainText: String, addNewEntryText: String) : ListTableWithButtons<T>() {
```

Comment:
The name is really confusing, how about `LogGroupSelectionTable`? Also why is this an abstract class with only one implementation?